.pre-commit-config.yaml

```diff
@@ -9,3 +9,8 @@ repos:
     hooks:
     - id: flake8
       language_version: python3
+-   repo: https://github.com/casperdcl/sh
+    rev: master
+    hooks:
+    - id: shfmt
+      language: golang
```

Comment:
why is it golang though? :)

Reply:
requirement by `shfmt` (suggested from https://github.com/iterative/dvc/issues/2834).

Will have to find something different if we don't want `golang`...

Reply:
btw on this point. Maybe don't have this as a pre-commit hook so we don't force devs to install `go`? We can keep it as a CI-only thing...

Reply:
nvm swapped to using `beautysh` which is python